src/examples/aes/Holmakefile.gen

```diff
@@ -1,6 +1,6 @@
 # includes
 # ----------------------------------
-INCLUDES   = ../.. lifter wp
+DEPENDENCIES   = lifter wp ../..
```

Comment:
since you added the referenced holheap directory into the includes, we can drop references like the `../..` here

Reply:
```suggestion
DEPENDENCIES = lifter wp
```
Right! I just didn't look into `examples/aes` files :sweat_smile:

Reply:
:)